diff --git a/app/models/account.rb b/app/models/account.rb
index fc7359cfc..c12961867 100644
--- a/app/models/account.rb
+++ b/app/models/account.rb
@@ -92,7 +92,7 @@ class Account < ApplicationRecord
   validates_with UnreservedUsernameValidator, if: -> { local? && will_save_change_to_username? && actor_type != 'Application' }
   validates :display_name, length: { maximum: 30 }, if: -> { local? && will_save_change_to_display_name? }
   validates :note, note_length: { maximum: 500 }, if: -> { local? && will_save_change_to_note? }
-  validates :fields, length: { maximum: 4 }, if: -> { local? && will_save_change_to_fields? }
+  validates :fields, length: { maximum: 8 }, if: -> { local? && will_save_change_to_fields? }

   scope :remote, -> { where.not(domain: nil) }
   scope :local, -> { where(domain: nil) }
@@ -324,7 +324,7 @@ class Account < ApplicationRecord
     self[:fields] = fields
   end

-  DEFAULT_FIELDS_SIZE = 4
+  DEFAULT_FIELDS_SIZE = 8

   def build_fields
     return if fields.size >= DEFAULT_FIELDS_SIZE

